### PR TITLE
Add event count on header

### PR DESCRIPTION
**Pull request by Warhedgehog**
This adds the count of event on each event category header (i.e. current, previous and upcoming)

### CODE REVIEW

**Review by Tesa27 — DISMISSED** [review]
LGTM

**Review by zoemarin — DISMISSED** [review]

**Review by ahominal — APPROVED** [review]
Looks good for me !

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/EventListAdapter.java [hunk 1]
```diff
@@ -119,6 +121,10 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
       }
       expandIcon.setRotation(expanded[eventCategory.ordinal()] ? 180f : 0f);
 
+      String numberOfEventsInCategory =
+          "(".concat(String.valueOf(eventsMap.get(eventCategory).size())).concat(")");
```
Comment (GabrielFleischer) [inline fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/EventListAdapter.java:125]: Wouldn't it be simpler to write this ?
```suggestion
      String numberOfEventsInCategory = "(" + eventsMap.get(eventCategory).size() + ")";
```

**Review by GabrielFleischer — DISMISSED** [review]
LGTM, I just added a question about a possible simplification.

You can implement it if you think it is pertinent.

**Review by GabrielFleischer — DISMISSED** [review]
LGTM

**Pull request merged**